app/api/search/documentQueryBuilder.js

```diff
@@ -353,14 +353,16 @@ export default function () {
         );
       });
       // suggested has an implied '__' as a prefix
-      properties.forEach(property => {
-        baseQuery.aggregations.all.aggregations[`__${property.name}`] = propertyToAggregation(
-          property,
-          dictionaries,
-          baseQuery,
-          true
-        );
-      });
+      if (clasificationIsEnabled) {
```

Comment:
Again, this comment is commenting now an `if`.

Reply:
Moved in previous commit.